commit b5b4d854d9d83127c7082f548400ce23bbc0faef (refs/changes/59/5159/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-11-28T22:00:43+00:00 (10 months ago)
    
    OS-7398 topo_usb_port_properties() doesn't properly handle ports with no protocol

diff --git a/usr/src/lib/fm/topo/modules/common/usb/topo_usb.c b/usr/src/lib/fm/topo/modules/common/usb/topo_usb.c
index ba578d6b3d..5a5c526c51 100644
--- a/usr/src/lib/fm/topo/modules/common/usb/topo_usb.c
+++ b/usr/src/lib/fm/topo/modules/common/usb/topo_usb.c
@@ -1211,7 +1211,7 @@ topo_usb_port_properties(topo_mod_t *mod, tnode_t *tn, topo_usb_port_t *port)
 			vers = "3.1";
 			break;
 		default:
-			break;
+			continue;
 		}
 
 		/*
